database/factories/PaymentFactory.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace dnj\Invoice\Database\Factories;
+
+use dnj\Account\Models\Transaction;
+use dnj\Invoice\Contracts\PaymentStatus;
+use dnj\Invoice\Models\Invoice;
+use dnj\Invoice\Models\Payment;
+use dnj\Number\Number;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class PaymentFactory extends Factory
+{
+    protected $model = Payment::class;
+
+    public function definition()
+    {
+        // TODO: Implement definition() method.
+        return [
+            'invoice_id' => Invoice::factory(),
+            'transaction_id' => null,
+            'method' => fake()->sentence(3),
```

Comment:
it's should be class name which implemented `Contracts\IPaymentMethod` interface

Reply:
I can't implement it